### ISSUE TITLE
Shut down this repo.

### ISSUE THREAD

**Issue by mikeal** [nodejs/logos#34]
This repo is really old and from the io.js days.

**Comment by zlatanvasovic** [comment]
I support this. It's needless.

**Comment by williamkapke** [comment]
+1

**Comment by bcomnes** [comment]
Whatever you end up doing, just don't delete the work thats there.  Move it to some archive org or something for historical purposes.

**Comment by mikeal** [comment]
> Whatever you end up doing, just don't delete the work thats there. Move it to some archive org or something for historical purposes.

Good point. Maybe rename it to something like "archive-iojs-logos" and make the readme clear.

**Comment by zlatanvasovic** [comment]
Or put it all in already-archived iojs repo?

**Comment by mikeal** [comment]
The main iojs repo was renamed to `node` and is the primary repo for Node.js now :)

**Comment by zlatanvasovic** [comment]
@mikeal Sorry, I meant iojs<b>.org</b> repository. ([this one](https://github.com/nodejs/iojs.org))
